docs/docs/installation.md

```
@@ -0,0 +1,89 @@
+- Node ≥ 16 recommended (top-level `await`, best WASM performance)
+- For browser use: modern browser with ES module and WebAssembly support
+
+:::tip CDN
+You can also load Img2Num directly from jsDelivr:
+
+```html
+<script type="module">
+  import { imageToUint8ClampedArray, bilateralFilter, kmeans, findContours }
+    from "https://cdn.jsdelivr.net/npm/img2num/build-wasm/index.js";
+</script>
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)